src/styles/block/card.scss

```diff
@@ -0,0 +1,96 @@
+.card__button {
+  display: flex;
+  justify-content: center;
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 16px;
+  color: rgb(255, 255, 255);
+  align-items: center;
+  cursor: pointer;
+  text-decoration: none;
+  border: 1px solid transparent;
+  background-color: var(--acent-color);
+}
+
+.card__button:hover {
+  border: 1px solid #00acdc;
+  background-color: #fff;
+  color: var(--acent-color);
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 32px 16px 16px;
+  width: 200px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-size: 14px;
+  font-weight: 400;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  color: var(--text-color-black);
+}
```

Comment:
As you use scss rewrite your blocks according to this `&__button {..}`

Reply:
done